opendistro-elasticsearch-index-management.release-notes.md

```diff
@@ -1,6 +1,14 @@
-## Version 1.3.0.0
+## Version 1.3.0.1 (Current)
 
-## New Features
+### Bug Fixes
+
+* Fixes rollover not checking all conditions
```

Comment:
Can you add PRs to each changes? Alerting and Job-scheduler are following this pattern. 

https://github.com/opendistro-for-elasticsearch/alerting/blob/master/opendistro-elasticsearch-alerting.release-notes.md

https://github.com/opendistro-for-elasticsearch/job-scheduler/blob/master/opendistro-elasticsearch-job-scheduler.release-notes